src/stores/canvas.ts

```diff
@@ -36,6 +40,14 @@ const useStoreCanvas = defineStore({
       })
       await promise
     },
+    resetCanvas() {
+      // delete
+      useStoreLine().deleteLines()
+      useStoreText().deleteTexts()
+      useStoreImage().deleteImages()
+      // save history
+      useStoreHistory().handleEventEndSaveHistory()
+    },
```

Comment:
```suggestion
    resetCanvas() {
      // delete
      useStoreLine().deleteLines()
      useStoreText().deleteTexts()
      useStoreImage().deleteImages()
      // reset transformer
      useStoreTransformer().$reset()
      // save history
      useStoreHistory().handleEventEndSaveHistory()
    },
```
transformerのリセットをお願いします。